xtask/src/internal.rs

```diff
@@ -263,6 +263,15 @@ pub struct RunCargoFuzz {
     pub args: Vec<String>,
 }
 
+#[derive(Parser, Clone, Debug)]
+pub struct BuildBaremetalVariantsOpt {
+    #[clap(
+        long,
+        help = "name of a specific baremetal variant (bios, or crosvm). If not specified, builds all variants."
```

Comment:
Based on the rest of the changes, I assume "bios" -> "qemu"

Reply:
Thanks. Fixed.